litellm/integrations/langfuse/langfuse.py

```diff
@@ -654,6 +655,13 @@ def _log_langfuse_v2(  # noqa: PLR0915
                         "completion_tokens": _usage_obj.completion_tokens,
                         "total_cost": cost if self._supports_costs() else None,
                     }
+                    usage_details = {
```

Comment:
Can you make this a typed dict ?

Reply:
Done in commit [d47ad69a1b2d006fbc15acea12bc47cc9d330bc1](https://github.com/BerriAI/litellm/pull/10955/commits/d47ad69a1b2d006fbc15acea12bc47cc9d330bc1)!

Reply:
please use the typed dict here usage_details - LangfuseUsageDetails(input=.., output=...)

Reply:
I was getting lit errors when doing that:

`integrations/langfuse/langfuse.py:662: error: Name "usage_details" already defined on line 645  [no-redef]`

Reply:
Any suggestion on how to do it without triggering the lint error?

Reply:
Never mind, I think I fixed it!!